taskiq/scheduler/scheduled_task.py

```diff
@@ -1,11 +1,12 @@
 import uuid
-from dataclasses import dataclass, field
+from dataclasses import field
```

Comment:
Here you should use pydantic's Field, In the ScheduledTask class need to be replaced with Field.

Reply:
True.

Reply:
Also, I've missed support for different pydantic versions. Now if should be fine.